test/hotspot/jtreg/gc/stress/gcold/TestGCOldWithShenandoah.java

```diff
@@ -1,5 +1,6 @@
 /*
 * Copyright (c) 2016, 2020, Oracle and/or its affiliates. All rights reserved.
+* Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
```

Comment:
General comment: Looking at the history, I might have expected RedHat copyright headers for many of these tests, but that isn't a change that's happened with generational shenandoah. So, nothing for us to do in this PR.